examples/rate-wars/.env.production

```
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_API_URL=https://demo-game.stg.env.bf-app.ch/api/graphql
+NEXT_PUBLIC_APP_URL=https://demo-game.stg.env.bf-app.ch
+NEXTAUTH_URL=https://demo-game.stg.env.bf-app.ch
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXTAUTH_URL key should go before the NEXT_PUBLIC_API_URL key

(UnorderedKey)